service/pod/framework.go

```diff
@@ -10,8 +10,8 @@ import (
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/giantswarm/kvm-operator/service/framework/podv2"
-	"github.com/giantswarm/kvm-operator/service/keyv2"
+	"github.com/giantswarm/kvm-operator/service/pod/keyv2"
+	"github.com/giantswarm/kvm-operator/service/pod/resourcesv2"
```

Comment:
There is no `framework` package. []framework.Resource are created in `resourcesv*`.